tests/topotests/bgp_features/test_bgp_features.py

```diff
@@ -150,7 +150,7 @@ def test_ospf_convergence():
         assert res is None, assertmsg
 
 
-def test_bgp_convergence():
+def xxxtest_bgp_convergence():
```

Comment:
Hm, do we need these changes? :)

Reply:
ouch - sorry, leftover from testing, will push a fix ...

Reply:
... should be fixed now